### PR TITLE
Adding test cases v3

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/HZM7iKQA/1294-add-unit-tests-for-access-survey-in-ras-frontstage

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ class DevelopmentConfig(Config):
 
     ACCOUNT_SERVICE_URL = os.getenv("ACCOUNT_SERVICE_URL", "http://localhost:8082/surveys/todo")
     ACCOUNT_SERVICE_LOG_OUT_URL = os.getenv("ACCOUNT_SERVICE_LOG_OUT_URL", "http://localhost:8082/sign-in/logout")
-    EQ_URL = os.getenv("EQ_URL", "http://localhost:8086/session?token=")
+    EQ_URL = os.getenv("EQ_URL", "http://localhost:8086/complete-eq?token=")
```
Comment (insacuri) [inline config.py:92]: why this change?  that's not the right url

Comment (creativeappdevamit) [reply config.py:92]: ohh.. needs to revert this.. this was not suppose to go in

**Review by insacuri — APPROVED** [review]
